lib/wikisnakker.rb

```diff
@@ -121,7 +121,7 @@ class Item
     def self.find(ids)
       lookup = Lookup.find(ids)
       data = lookup.values
-      inflated = data.map { |rd| new(rd) }
+      inflated = data.reject { |d| d[:missing] == '' }.map { |rd| new(rd) }
```

Comment:
I think it might be better doing this `reject` on the previous line. Here it makes it seem like it's something to do with the inflation, when really anything with this set isn't valid `data` in the first place.